extension/addreactionparticipants.js

```diff
@@ -0,0 +1,60 @@
+const addReactionParticipants = currentUser => {
+	$('.comment-reactions').each((index, reactionsContainer) => {
+		const $reactionsContainer = $(reactionsContainer);
+		if (!$reactionsContainer.hasClass('has-reactions')) {
+			return;
+		}
+
+		const $reactionButtons = $reactionsContainer.find('.comment-reactions-options .reaction-summary-item[aria-label]');
+
+		$reactionButtons.each((index, element) => {
+			element.addEventListener('click', () => {
```

Comment:
You mean one listener and checking if it matches the target class? It's not tough, but does it have a big benefit? I don't mind doing it, just curious.